### DISCUSSION TITLE
Sponsors unknowingly stop sponsoring

### DISCUSSION THREAD

**Discussion by borkdude** [community/community#15446] [Other Feature Feedback, Questions, & Ideas]
Thanks for providing Github Sponsors. It allows me to make a living from OSS. Because Github Sponsors is now an important source of income for me, I wanted to share the following feedback.

It happened to me several times in the past few months that sponsors cancelled. But when I asked them why they cancelled, they did not even know this. In one case it was because of a credit card expiration, but it also seems to happen randomly. Luckily most sponsors restore their sponsoring, but I'm sure I have lost a few along the way due to this behavior.

Perhaps Github can more pro-actively notify users when they stop sponsoring someone without their explicit consent (regardless of reason, credit card expiration, etc.). This would help keeping the stream of income for developers who rely on it for their daily bread more reliable.

Literal quotes from sponsors:

> Hey! Just realized I had stopped sponsoring you. Not sure how/when that happened. Back to sponsoring now!

> I haven’t noticed that my GitHub payment was failing so it seems I stopped being a sponsor a month ago :man-facepalming: (

> Huh... It’s card expiration I’d tell you if intentional

**Comment by teodorlu** [comment]
This literally just happened to me, not sure why/how.

Possibly my fault, but I definitely didn't make a conscious decision to stop sponsoring @borkdude.

**Comment by pedrorgirardi** [comment]
It happened to me too. 🫤

I had a problem with my billing, and after I fixed the problem I was no longer sponsoring @borkdude . I didn't notice that I stopped sponsoring him for maybe a few months.

**Comment by jlord** [comment]
Thanks for reporting this! I'll write it up for the team, definitely want to improve this experience.

**Reply by KalleVuorjoki** [reply]
Unfortunately this is still a relevant issue 😞

**Comment by ghost** [comment]
It takes a long process for me to get it. I also wonder if, in the next 5 years, I will achieve it or not. My project is small, and you are lucky to live by doing what you love. I hope it gets better every day. If, in the next 5 years, my little project has 100,000 stars and Google uses it—who knows. But all I know now is that I'm still happy doing what I love, whether I'm sponsored or not. Wish you a great day.
